app/controllers/lists_controller.rb

```diff
@@ -15,6 +15,10 @@ def create
     end
   end
 
+  def index
+    @lists = List.all
```

Comment:
this should be paginated or limited to ensure the collection of lists do not exceed memory limits.